test/Vagrantfile

```diff
@@ -22,18 +23,24 @@ $CPU = (ENV['CPUS'] || "2").to_i
 if ENV['NETNEXT'] == "true"
     $SERVER_BOX = $NETNEXT_SERVER_BOX
     $SERVER_VERSION = $NETNEXT_SERVER_VERSION
+    $CHECK_GUEST_ADDITIONS = false
```

Comment:
Please add a comment that says that the guest additions are already installed in the box image and include a link to the packer ci scripts for clear reference.